src/Value/Lines.php

```diff
@@ -74,20 +78,31 @@ public function rewind(): void
      */
     public function seek($line): void
```

Comment:
@smoench should we use `seek(0)` in `rewind()`?

Reply:
this will introduce new behavior ( exception is thrown when rewinding an empty iterator ), but I don't think it's necessary.